### PR TITLE
removed sentences and added correct sentences

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #10054 , so i corrected the wrong sentences to correct sentences. i added after verifying with Windows 11 build number 22000.318  GPO.

Screenshot 2021-11-11 003343

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-userrights.md [hunk 1]
```diff
@@ -1080,9 +1080,10 @@ GP Info:
 
 <!--/Scope-->
 <!--Description-->
-This security setting determines which service accounts are prevented from registering a process as a service.
+This security setting determines which users are prevented from logging on at the computer. This policy setting supersedes the Allow log on locally policy setting if an account is subject to both policies.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-userrights.md:1083]:
```suggestion
This security setting determines which users are prevented from logging on at the computer. This policy setting supersedes the **Allow log on locally** policy setting if an account is subject to both policies.
```

File: windows/client-management/mdm/policy-csp-userrights.md [hunk 2]
```diff
@@ -1080,9 +1080,10 @@ GP Info:
...
+
 > [!NOTE]
-> This security setting does not apply to the System, Local Service, or Network Service accounts.
+> If you apply this security policy to the Everyone group, no one will be able to log on locally.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-userrights.md:1086]:
```suggestion
> If you apply this security policy to the **Everyone** group, no one will be able to log on locally.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RAJU2529, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @RAJU2529

**Pull request merged**